azext_iot/sdk/digitaltwins/dataplane/operations/import_jobs_operations.py

```diff
@@ -0,0 +1,354 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+import uuid
+from msrest.pipeline import ClientRawResponse
+
+from .. import models
+
+
+class ImportJobsOperations(object):
+    """ImportJobsOperations operations.
+
+    :param client: Client for service requests.
+    :param config: Configuration of service client.
+    :param serializer: An object model serializer.
+    :param deserializer: An object model deserializer.
+    :ivar api_version: The requested API version. Constant value: "2021-06-30-preview".
```

Comment:
lets try to keep one api version throughout the sdk - can you change the other operations/files to use the newer api too?

Reply:
jk everything is using this api